### ISSUE TITLE
No ability to send dynamic_template_data

### ISSUE THREAD

**Issue by pod2metra** [sklarsa/django-sendgrid-v5#41]
It's no ability to send dynamic_template_data via package that was installed via pip. Please update version via publish script.

**Comment by sklarsa** [comment]
Which versions of Django-sendgrid-v5 and sendgrid-python are you using and what errors are you getting?

**Comment by pod2metra** [comment]
```
pip freeze | grep sendgrid
django-sendgrid-v5==0.6.893
sendgrid==5.6.0```

**Comment by sklarsa** [comment]
Published to pypi version 0.7.0
